query-planner/tests/cucumber.rs

```diff
@@ -0,0 +1,102 @@
+use cucumber::cucumber;
+use std::fmt;
+
+type Result<T> = std::result::Result<T, QueryPlanError>;
+
+// stub out a custom QueryPlanError
+#[derive(Debug, Clone)]
+struct QueryPlanError;
+impl fmt::Display for QueryPlanError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "error in building query plan")
+    }
+}
+
+struct Schema {}
+fn parse_schema(_schema: &str) -> Result<Schema> {
+    Ok(Schema {})
+}
+
+struct Query {}
+fn parse_query(_query: &str) -> Result<Query> {
+    Ok(Query {})
+}
+#[derive(Debug)]
+struct QueryPlan {
```

Comment:
`crate::model::QueryPlan` (or maybe it has to be `apollo-query-planner::`, not sure)